app/lib/proxy_config_affecting_changes.rb

```diff
@@ -73,6 +73,19 @@ def flush
       @tracked_objects.clear
     end
 
+    def reported_clear
```

Comment:
Just to be sure, this is only required/relevant when `RAILS_ENV=test`, right?

Reply:
See commit message of 6a508b48bc0c3ac45e8b45aab7f66c6d9c250a2a